engine/execution/computation/manager.go

```diff
@@ -231,12 +231,9 @@ func DefaultFVMOptions(chainID flow.ChainID, cadenceTracing bool, extensiveTraci
 			reusableRuntime.NewReusableCadenceRuntimePool(
 				ReusableCadenceRuntimePoolSize,
 				runtime.Config{
-					TracingEnabled:        cadenceTracing,
-					AccountLinkingEnabled: true,
```

Comment:
is account linking enabled by default going forward?

Reply:
Account linking got replaced by account capability controllers, which are not behind behind a feature flag and always enabled.